packages/agents-activity/src/conversation/membershipSourceTypes.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright(c) Microsoft Corporation.All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { z } from 'zod'
+
+/**
+ * Enum defining the type of roster the user is a member of.
+ */
+export enum MembershipSourceTypes {
+  /**
+   * The user is a direct member of the current channel.
```

Comment:
Doc update request after discussion with Anand:
```suggestion
   * The source is that of a channel and the user is a member of that channel.
```

Reply:
Please let me know if these are confusing? The idea is to avoid using `direct` when describing source type. We discussed it could be possible in the future to share a channel with another channel, and in that case a description of `direct` wouldn't fit.